Gemfile.base

```diff
@@ -79,6 +79,8 @@ when 'mysql'
   gem 'mysql2', '~>0.3.11'
 end
 
+gem '3scale_client', '~> 2.6.1', require: false
```

Comment:
Oh so it was not tested neither is working in 2.2 (well we never committed to include it anyway)
The seed is already done: https://github.com/3scale/porta/blob/master/db/seeds.rb#L102-L104
But using internal API of MT is not reported to backend.
It would be good to add it, even if not use so it will simplify our code

/cc @3scale/product if that makes sense to make it a feature for 2.4

Reply:
I don't understand this comment. This dependency was there and just changes its' place.

Reply:
Well it was not in on-premises.

Reply:
But fine for me now.